argocd_agent_installation/argocd-agent-installation.adoc

```diff
@@ -0,0 +1,67 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="argocd-agent-installation"]
+= Installing Argo CD Agent
+:context: argocd-agent-installation
+
+toc::[]
+
+The Argo CD Agent consists of two components, *Principal* and *Agent* that work together to synchronize applications between the control plane and workload clusters in a hub-and-spoke configuration. This guide provides instructions for installing the required Argo CD Agent components. You can install the Principal component by using Argo CD resources and the Agent component by using Helm charts.
+
+For an architectural overview and more details about Argo CD Agent, see the `Understanding Argo CD Agent architecture` section.
+
+// Argo CD Agent Terminologies
+include::modules/gitops-argocd-agent-terminologies.adoc[leveloffset=+1]
+
+.Prerequisites
+
+* You have installed the {gitops-title} Operator on two {OCP} clusters.
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the {gitops-title} `argocd` CLI.
+* You have logged in to the default Argo CD instance.
+* You have installed a `cluster-scoped Argo CD` instance in your defined namespace.
```

Comment:
As Jonathan mentioned in one of his comment, steps to enable cluster-scoped instance should be provided, you can add link to operator doc here. Also links for `Apps in any namespace`, `argocd-agentctl`, `argocd CLI` etc can be provided here, my draft doc has links to all these or you can give openshift-gitops doc links for these.

Reply:
Got it, then Additional Resources is also Okay with me.